examples/scaling-criteo/01-Download-Convert.ipynb

```diff
@@ -1,304 +1,304 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    BASE_DIR = os.environ.get("BASE_DIR", "/raid/data/criteo")
</pre><p>it is unlikely that everyome has a <code>/raid</code> directory. instead of falling through, this should prompt the user to set <code>BASE_DIR</code> to a location that satisfies the requirements.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>